extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -266,7 +266,9 @@ export class ResourceTypePickerDialog extends DialogBase {
 			this._agreementCheckboxChecked = false;
 			this._agreementContainer.clearItems();
 			this._optionsContainer.clearItems();
-			this.updateToolsDisplayTable();
+			this._toolsLoadingComponent.loading = false;
+			this._toolsTable.data = [[localize('deploymentDialog.NoRequiredTool', "No tools required"), '']];
```

Comment:
Please include a screenshot - I'm not sure how this sentence makes sense when there's no options to even select

Reply:
![image](https://user-images.githubusercontent.com/6816294/95637110-c6d80880-0a45-11eb-8bc1-0398e62967aa.png)

Reply:
I don't like having an empty table there to begin with - but removing it is going to annoying I think. 

If you don't have a row what does it look like? That'd be slightly preferable I think.

Reply:
![image](https://user-images.githubusercontent.com/6816294/95638728-8464fa80-0a4a-11eb-98fa-53bde687852b.png)

Yeah, this looks better